stream-chat-android-client/src/main/java/io/getstream/chat/android/client/helpers/QueryChannelsPostponeHelper.kt

```diff
@@ -27,11 +28,11 @@ internal class QueryChannelsPostponeHelper(
         channelId: String,
         request: QueryChannelRequest
     ): Call<Channel> = CoroutineCall(coroutineScope) {
-        doSafeJob { api.queryChannel(channelType, channelId, request) }.execute()
```

Comment:
This was performing a blocking operation inside a suspending function, which is never a good idea, see for example https://elizarov.medium.com/blocking-threads-suspending-coroutines-d33e11bf4761#4c8b